### PR TITLE
refactor(OnboardingRepeatPin): code

### PR DESCRIPTION
**Pull request by samandarkuchkarov**
Fixes #

## Proposed Changes

-Change the imports to absolute:
-Move text to i18n
-Change the colors from the constants to similar ones in Color
-Transferring the screen logic to components
-Replace navigation hooks with typed navigation hooks in folder src/hooks

## Checklist

- [x] I have read the rules of [**contribution**](https://github.com/haqq-network/haqq-wallet/blob/main/docs/contribution.md#important-steps).

### CODE REVIEW

**Review by gHashTag — CHANGES_REQUESTED** [review]
Fix comment

File: src/app.tsx [hunk 1]
```diff
@@ -208,7 +208,9 @@ export const App = () => {
         <TransactionsContext.Provider value={transactions}>
           <WalletsContext.Provider value={wallets}>
             <NavigationContainer ref={navigator} theme={navTheme}>
-              <Stack.Navigator screenOptions={basicScreenOptions}>
+              <Stack.Navigator
+                initialRouteName="ledger"
```
Comment (gHashTag) [inline src/app.tsx:212]: Why `ledger`?

Comment (samandarkuchkarov) [reply src/app.tsx]: done

**Review by gHashTag — APPROVED** [review]

File: src/screens/ledger.tsx [hunk 2]
```diff
@@ -27,7 +27,9 @@ const screenOptionsBiometry: ScreenOptionType = {title, headerBackHidden: true};
 export const LedgerScreen = () => {
   const user = useUser();
   return (
-    <LedgerStack.Navigator screenOptions={popupScreenOptions}>
+    <LedgerStack.Navigator
+      initialRouteName="onboardingRepeatPin"
```
Comment (vivalaakam) [inline src/screens/ledger.tsx:31]: why?

File: src/components/onboarding-repeat-pin.tsx [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+            navigation.navigate('onboardingBiometry', {
+              ...params,
+              biometryType: app.biometryType,
+            });
+          } else {
+            navigation.navigate(nextScreen ?? 'signupStoreWallet', {
+              ...params,
+            });
+          }
+        });
```
Comment (vivalaakam) [inline src/components/onboarding-repeat-pin.tsx:46]: move it on screen layer

**Review by vivalaakam — CHANGES_REQUESTED** [review]
why [ios/haqq/haqq.entitlements](https://github.com/haqq-network/haqq-wallet/pull/578/files#diff-d277b486442c6aa3b3a287d2d3d118dd187b9eb1325321010173170cc4f7904e) is removed???

**Comment by vivalaakam** [comment]
and for these reason you removed file which you don't know?

**Review by gHashTag — CHANGES_REQUESTED** [review]
Recovery entitlements and fix comments

File: src/components/onboarding-repeat-pin.tsx (hunk 3 above)
Comment (samandarkuchkarov) [reply src/components/onboarding-repeat-pin.tsx]: done

**Review by gHashTag — APPROVED** [review]

File: src/screens/ledger.tsx [hunk 4]
```diff
@@ -27,7 +27,9 @@ const screenOptionsBiometry: ScreenOptionType = {title, headerBackHidden: true};
 export const LedgerScreen = () => {
```
Comment (vivalaakam) [inline src/screens/ledger.tsx:27]: Revert this file pls. it is not related to the main task

Comment (samandarkuchkarov) [reply src/screens/ledger.tsx]: done

**Review by vivalaakam — APPROVED** [review]

File: src/i18n.ts [hunk 5]
```diff
@@ -61,6 +61,7 @@ export enum I18N {
   transactionAddressPlaceholder,
   transactionAddressError,
   transactionAddressLabel,
+  TransactionEmpty,
```
Comment (gHashTag) [inline src/i18n.ts:64]: `camelCase`

**Pull request merged**